library/templates/compare-xml.yaml

```diff
@@ -0,0 +1,67 @@
+- job-template:
```

Comment:
IMHO: this file is stash related, so its name or location should reflect this fact.